notebooks/env.ipynb

```diff
@@ -0,0 +1,628 @@
+{
```

Comment:
<pre class="ql-syntax" spellcheck="false">Line #1.    !pip install ruamel.yaml
</pre><p>I don't think this is actually necessary, let me know if it is. </p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/chandar-lab/RLHive/pull/305/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='6'/>